modules/services/window-managers/i3-sway/i3.nix

```
@@ -185,6 +185,7 @@ let
       ${concatStringsSep "\n" (map floatingCriteriaStr floating.criteria)}
       ${concatStringsSep "\n" (map windowCommandsStr window.commands)}
       ${concatStringsSep "\n" (map startupEntryStr startup)}
+      ${concatStringsSep "" (map workspaceOutputStr workspaceOutputAssign)}
```

For consistency, maybe do this:
```suggestion
      ${concatStringsSep "\n" (map workspaceOutputStr workspaceOutputAssign)}
```
and update the `workspaceOutputStr` function below.